sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2SQLSuite.scala

```diff
@@ -1841,6 +1841,22 @@ class DataSourceV2SQLSuite
     }
   }
 
+  test("DeleteFrom: should refresh caches referencing the table") {
```

Comment:
Shall we add a JIRA prefix here @sunchao?

Reply:
Good idea. Will do.